swagger-docs.md

```diff
@@ -0,0 +1,143 @@
+# Goede documentatie in Swagger files
+
+Als je API klaar is, heb je - als alles goed is - een Swagger file. Soms wordt deze gegenereerd, soms met de hand gemaakt. Belangrijk is dat deze Swagger file goed is gedocumenteerd. Een operatie `GET /Assets` met als omschrijving *"Gets the assets"* is vrijwel nutteloos. We merken nochtans, dat veel gegenereerde Swagger files z'n resultaat opleveren. Vandaar, dat we je graag enkele tips geven hoe dit beter kan.  
```

Comment:
De url /Assets is ook niet API requirements conform alles moet lowercase

Reply:
... z**o** 'n resultaat opleveren. Vandaar dat we ...